clouddrift/adapters/gdp.py

```diff
@@ -19,7 +19,9 @@
 
 GDP_VERSION = "2.00"
```

Comment:
We're moving to version 2.01 so how to make this changeable?

Reply:
> 



> This worked for 100 drifters and I also tried for the full dataset. The downloading steps indicates 19284 files but the final ragged array had `traj=19082`.
> 
> <img alt="Screenshot 2023-02-10 at 2 32 27 AM" width="759" src="https://user-images.githubusercontent.com/7576346/218030127-2c2208c7-d83f-4323-887f-589907c4f7a1.png">
> 
> What is the discrepancy?

My only guess is if two files have different names and their IDs are the same, you would lose 1 file in the ragged array.

Reply:
``
d
d
``

Reply:
I did a naive search for this issue and I can confirm that there are 19284 unique field `ID` in the 19284 files. So the issue must be from somewhere else?